surya/model/layout/model.py

```diff
@@ -0,0 +1,27 @@
+import torch
+
+from surya.model.detection.model import EfficientViTForSemanticLayoutSegmentation
+from surya.model.detection.config import EfficientViTConfig
+from surya.model.detection.processor import SegformerImageProcessor
+from surya.settings import settings
+
+def load_model(checkpoint=settings.DETECTOR_MODEL_CHECKPOINT, device=settings.TORCH_DEVICE_MODEL, dtype=settings.MODEL_DTYPE) -> EfficientViTForSemanticLayoutSegmentation:
```

Comment:
Please update the docs to indicate how to import the layout model - I think we also have to update the ocr app and cli scripts, right?  This also needs corresponding changes in marker, since it loads the layout model

Reply:
Updated!